CONTRIBUTING.md

```diff
@@ -84,7 +84,6 @@ For example, APM Server version `7.15.0` corresponds with the Go APM Agent versi
 
 1. Update `version.go` and `internal/apmversion/version.go`, and then run `make update-modules`
 2. Update [`CHANGELOG.asciidoc`](changelog.asciidoc), by adding a new version heading (`==== 1.x.x - yyyy/MM/dd`) and changing the base tag of the Unreleased comparison URL
-3. For major and minor releases, update the EOL table in [`upgrading.asciidoc`](docs/upgrading.asciidoc).
```

Comment:
Since https://github.com/elastic/apm-agent-go/pull/1335, there is no EOL table anymore.